packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -38,6 +38,18 @@ typedef DragTargetAcceptWithDetails<T> = void Function(DragTargetDetails<T> deta
 /// Used by [DragTarget.builder].
 typedef DragTargetBuilder<T> = Widget Function(BuildContext context, List<T> candidateData, List<dynamic> rejectedData);
 
+/// Signature for building children of a [DragTarget].
+///
+/// The `candidateDetails` argument contains the list of [DragTargetDetails] for
+/// [Draggable]s hovering over this [DragTarget] and that has passed
+/// [DragTarget.onWillAccept]. The `rejectedDetails` argument contains the list
+/// of [DragTargetDetails] for [Draggable]s hovering over this [DragTarget]
+/// that will not be accepted by the [DragTarget].
+///
+/// Used by [DragTarget.detailsBuilder].
+typedef DragTargetDetailsBuilder<T> =
```

Comment:
Is it better to call it [DragTargetDetailedBuilder] (as well as other related symbols)?